From b41a93be4b5608996b434a1d674df32a44963c7f Mon Sep 17 00:00:00 2001
From: schnoog <schnoog@schnoog.eu>
Date: Wed, 24 Jan 2024 17:23:09 +0100
Subject: [PATCH 27/43] fix #10 Added #include "autoconf.h" so src/dirzctl.c
 and src/hx711s.c Without it CONFIG_CLOCK_FREQ is unknown to both files and
 compiling of additional MCUs (f.e. RP2040) fail. Changes tested with V3SE
 (STM32) and RP2040 as micro-controller architecture.

---
 src/dirzctl.c | 1 +
 src/hx711s.c  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/dirzctl.c b/src/dirzctl.c
index 3692240c1..befc27978 100755
--- a/src/dirzctl.c
+++ b/src/dirzctl.c
@@ -1,3 +1,4 @@
+#include "autoconf.h" // CONFIG_CLOCK_FREQ
 #include "basecmd.h"        // oid_alloc
 #include "board/gpio.h"     // struct gpio_in
 #include "board/irq.h"      // irq_disable
diff --git a/src/hx711s.c b/src/hx711s.c
index ea64dd298..fd1caa9a1 100755
--- a/src/hx711s.c
+++ b/src/hx711s.c
@@ -1,3 +1,4 @@
+#include "autoconf.h" // CONFIG_CLOCK_FREQ
 #include "basecmd.h"        // oid_alloc
 #include "board/gpio.h"     // struct gpio_in
 #include "board/irq.h"      // irq_disable
-- 
2.39.5 (Apple Git-154)

